packages/client/components/RetroReflectPhase/PhaseItemEditor.tsx

```diff
@@ -272,6 +283,9 @@ const PhaseItemEditor = (props: Props) => {
                   editorState={card.editorState}
                   readOnly
                   disableAnonymity={disableAnonymity}
+                  teamId={teamId}
+                  editorRef={editorRef}
```

Comment:
-1 this part seems wrong. If we have some cards in flight, they should not need a reference to the editor anymore as I should not be able to do any edits anymore